substrate/frame/identity/src/migration.rs

```diff
@@ -92,7 +115,7 @@ pub mod v1 {
 			let mut interrupted = false;
 
 			for (account, registration) in v0::IdentityOf::<T>::iter() {
-				IdentityOf::<T>::insert(account, (registration, None::<Username<T>>));
+				vx::IdentityOf::<T>::insert(account, (registration, None::<Username<T>>));
```

Comment:
```suggestion
				v1::IdentityOf::<T>::insert(account, (registration, None::<Username<T>>));
```
Otherwise it migrates to v2.

Reply:
Refactored.